README.md

```diff
@@ -1,4 +1,10 @@
 # degitx-simulator
 Simulation system for DeGItX
 
+[![Build Status](https://github.com/cqfn/degitx-simulator/workflows/Build%20degitx-simulator/badge.svg)](https://github.com/cqfn/degitx-simulator)
+![Build white paper document](https://github.com/cqfn/degitx-simulator/workflows/Build%20white%20paper%20document/badge.svg)
```

Comment:
@IlyaMoskva maybe we can add a link from this badge to WP document uploaded? https://central.artipie.com/degit/wp/simulator-latest.pdf

Reply:
@g4s8, I did it in a same way as all cqfn projects. Moreover, next line we have link with description what is pretty clean.